packages/react-server/src/ReactServerStreamConfigNode.js

```diff
@@ -33,46 +34,101 @@ export function flushBuffered(destination: Destination) {
   }
 }
 
+const VIEW_SIZE = 2048;
+let currentView = null;
```

Comment:
How does this account for concurrent requests on the same server? Won't the state be shared?

Reply:
I guess begin and complete always get called within the same synchronous transaction

Reply:
Yeah we could put it on the request but begin always resets state (in case of a throw) and complete is always called sync after each begin otherwise so it ends up always being synchronized